livekit-ffi/src/conversion/room.rs

```diff
@@ -98,6 +98,7 @@ impl From<DisconnectReason> for proto::DisconnectReason {
             DisconnectReason::UserUnavailable => Self::UserUnavailable,
             DisconnectReason::UserRejected => Self::UserRejected,
             DisconnectReason::SipTrunkFailure => Self::SipTrunkFailure,
+            DisconnectReason::ConnectionTimeout => Self::ConnectionTimeout,
```

Comment:
Can you create a similar conversion for the TrackFeature?
`impl From< AudioTrackFeature > for proto:: AudioTrackFeature {`

We're currently only relying on the integer, which is unsafe and will panic as soon as the proto adds a new one